src/sentry/workflow_engine/handlers/condition/event_frequency_handlers.py

```diff
@@ -71,15 +77,15 @@ class EventFrequencyCountHandler(EventFrequencyConditionHandler, DataConditionHa
     }
 
     @staticmethod
-    def evaluate_value(value: WorkflowJob, comparison: Any) -> DataConditionResult:
-        if len(value.get("snuba_results", [])) != 1:
+    def evaluate_value(value: WorkflowEvaluationData, comparison: Any) -> DataConditionResult:
+        if not ((data := value.get("data")) and isinstance(data, list) and len(data) == 1):
```

Comment:
NOTE: when evaluating all conditions a `WorkflowJob` will be passed to slow conditions, which actually take in `WorkflowEvaluationData`. i couldn't find a way around this to make mypy happy AND have the types actually be respected -- since `data` is not a key in `WorkflowJob` i need to check for its existence, if it doesn't exist then we can ignore the input